src/Test/Utilities/Portable/Metadata/ILValidation.cs

```diff
@@ -3,16 +3,175 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.IO;
 using System.Linq;
+using System.Reflection;
 using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Security.Cryptography;
 using System.Text;
+using System.Threading;
 using System.Xml;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis;
 using Microsoft.Metadata.Tools;
 
 namespace Roslyn.Test.Utilities
 {
     public static class ILValidation
     {
+        public static bool IsStreamSigned(Stream moduleContents)
```

Comment:
What type of signing is this checking for: full, public, etc ... Please add a doc comment explaining.